package.json

```diff
@@ -18,11 +18,12 @@
   },
   "homepage": "https://github.com/dwyl/tachyons-bootstrap#readme",
   "dependencies": {
-    "http-server": "^0.10.0",
-    "tachyons": "^4.7.4"
+    "http-server": "^0.10.0"
   },
   "devDependencies": {
+    "codecov": "^2.2.0",
```

Comment:
what is the difference between installing the `codecov` npm module and running the Bash script? 🤔